sdks/tutorial/index.md

```diff
@@ -1,5 +1,192 @@
+    note: 'Winter is coming'
+}))
+console.log(`Your new patient id : ${createdPatient.id}`);
+```
+
+When creating your patient, iCure will assign an id to it, format as an uuid.
+
+That's exactly what we'll need to retrieve the information of your patient. 
+
+# Get your newly created Patient information
+To retrieve your patient information, you will need to call the `patientApi.getPatient` service, providing
+the id iCure assigned to your patient during its creation. 
+
+<!-- file://code-samples/tutorial/doctor-centric-app/index.mts snippet:get your patient information-->
+```typescript
+const johnSnow = await api.patientApi.getPatient(createdPatient.id)
+expect(createdPatient.id).to.be.equal(johnSnow.id)
+```
+
+If you would like to know more about the information contained in Patient, go check the [References](../references/classes/Patient.md)
+
+Let's now create some medical data for our patient. 
+
+# Create your first data sample
+A data sample can be any piece of information related to the health of a patient. (Measures, symptoms, ...)
+
+When a patient goes to the doctor, the measures the doctor takes, the symptoms the patient is listing are considered 
+as data samples in iCure. 
+
+To create a new data sample, use the `dataSampleApi.createOrModifyDataSample` or `dataSampleApi.createOrModifyDataSamples`
+if you have multiple of them. 
+You can also link those medical data to the patient you just created through the `patientId`argument. 
+
+<!-- file://code-samples/tutorial/doctor-centric-app/index.mts snippet:create your patient first medical data-->
+```typescript
+const createdData = await api.dataSampleApi.createOrModifyDataSamplesFor(johnSnow.id, [new DataSample({
+    labels: new Set([new CodingReference({ type: 'LOINC', code: '29463-7', version: '2' })]),
+    content: { en: { numberValue: 92.5 } },
+    valueDate: 20220203111034,
+    comment: 'Weight'
+}),
+    new DataSample({
+        labels: new Set([new CodingReference({ type: 'LOINC', code: '8302-2', version: '2' })]),
+        content: { 'en': { numberValue: 187 } },
+        valueDate: 20220203111034,
+        comment: 'Height'
+    })
+])
+```
+
+When creating your data samples, iCure will assign an id to each of them as for patients.
+
+But what happens if you don't know these ids ? How can you find back your data ?
+
+
+# Search for data samples satisfying some criteria
+Sometimes in your application, you will have to find multiple data, satisfying some specific criteria. 
+Some examples : 
+- You want to find all medical data of your patient with id ...; 
+- You want to find all patients containing 'Smith' in their name; 
+- You want to find all your male patients born between 01-01-1970 and 31-12-1970;
+- ...
+
+For this, we'll use the `dataSampleApi.filterDataSamples` service, creating our own search criteria.
+Let's say we would like to find back all medical data of our previously created Patient, with the corresponding 
+LOINC Body Weight label. 
```

Comment:
The examples above mention searching for patients, but here we talk only about data sample filters, and it may be confusing: the user may think they have to use filterDataSamples also to search for patients with some criteria.
It may be better to give some general information and then the example specific for data samples.
```suggestion
You can do this using the `filter` services of the apis, such as `dataSampleApi.filterDataSamples` or `patientApi.filterPatients`.
Let's say we would like to find back all medical data with the LOINC label corresponding to the body weight for the patient we have created. 
```